setup.py

```diff
@@ -0,0 +1,13 @@
+from setuptools import setup
+
+setup(
+    name='xbundle',
+    version='0.1.0',
+    packages=['xbundle'],
+    scripts=['bin/xbundle_convert.py'],
+    author='Isaac Chuang, Shawn Milochik',
+    author_email='ichuang@mit.edu, milochik@mit.edu',
+    description='Converts edX courses between OLX and single-XML formats.',
+    url='https://github.com/mitodl/xbundle',
+    install_requires=['lxml>=3.4.4', 'BeautifulSoup>=3.2.1'],
```

Comment:
I tend to follow the advice from https://caremad.io/2013/07/setup-vs-requirement/ and not specify versions in the setup.py file, but do pin to exact versions in requirements.txt.  It would be nice to do that here, but I'm open to doing this.